acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/node/configuration/BesuNodeFactory.java

```diff
@@ -134,7 +141,7 @@ public BesuNode createNodeWithAuthentication(final String name)
         new BesuNodeConfigurationBuilder()
             .name(name)
             .jsonRpcEnabled()
-            .jsonRpcAuthenticationEnabled()
+            .jsonRpcAuthenticationEnabled("authentication/auth.toml")
```

Comment:
I would make `"authentication/auth.toml"` a constant ... easier to modify in future if required.